quickstart/configure-mayastor.md

```diff
@@ -15,12 +15,20 @@ A pool is defined declaratively, through the creation of a corresponding `DiskPo
 
 #### Permissible Schemes for `spec.disks` under DiskPool CR
 
+{% hint style="info" %}
+It is <b>highly recommended</b> to specify the disk using its `UUID_number` to ensure unique identification.
+The usage of `dev/sdx` might result in data corruption issues.
+To get the UUID_number of a disk, execute:
+`sudo blkid | grep UUID=`
```

Comment:
Is this the right way?
@rohan2794 can you please confirm?

Reply:
Also, we have noticed that in some cases, by-uuid is not available, in such scenarios, we need to go with the next best thing i.e. by-id

Reply:
Also I don't see the utility of grep UUID, at least on my box I see
```
[bdias@bdias-precisiontower5810 tmp]$ sudo blkid
[sudo] password for bdias: 
/dev/sdb4: UUID="2e679452-189e-4b93-bc65-18e34fe64625" BLOCK_SIZE="4096" TYPE="ext4" PARTUUID="0644fdd7-a678-42ac-aef1-0879a1a8fb5a"
/dev/sdb2: UUID="81259585-3026-4789-8c7b-0840ebe4a001" TYPE="swap" PARTUUID="170675e4-028f-4860-804c-30e48e8f8bcb"
/dev/sdb3: UUID="e1da6f07-7ff7-4f8d-9d8a-022976d985fe" BLOCK_SIZE="4096" TYPE="ext4" PARTUUID="3c434e9c-c9bb-4552-855e-25b9cee35dac"
```

Reply:
> Also I don't see the utility of grep UUID, at least on my box I see

Yes, that was my doubt as well. Need to provide the path/mapping for the user.

Reply:
Updated the text to a more generic one, with UUID_number as one of the ways.